.travis.yml

```diff
@@ -62,7 +36,7 @@ deploy:
     on:
       repo: dart-lang/site-www
       branch: master
-      condition: $TASK == *build*
+      condition: $TASK == *tool/build.sh* && $TRAVIS_DART_VERSION == stable && $TRAVIS_EVENT_TYPE == push
```

Comment:
I've started looking into CRON jobs builds for this site (for #1247). I've added an extra condition to ensure that cron jobs don't trigger a site deploy.